Automattic/wp-calypso
# Redirect Calypso Home to WooCommerce Home in Ecommerce plans

#### Proposed Changes

This PR aims to redirect the default home URL `wordpress.com/home/[site]` to `[site].wpcomstaging.com/wp-admin/admin.php?page=wc-admin` only on Ecommerce plans. This should be considered a temporary redirection until we create **a dedicated Home for Ecommerce** in Calypso.

As you might see, the redirection happens with a bit of delay, thus causing a flash-of-content. Could we implement this in a way that it's instant? 🤔 

#### Testing Instructions



* Apply this patch
* Select and visit an Ecommerce site from "My Sites"
* Notice the redirection to the WooCommerce Home page.

#### Pre-merge Checklist




Related to #68765
